core/state_transition.go

```diff
@@ -347,10 +349,16 @@ func (st *StateTransition) applyEditValidatorTx(ev *staking.EditValidator, block
 		return errValidatorNotExist
 	}
 	wrapper := st.state.GetStakingInfo(ev.ValidatorAddress)
+
+	oldRate := wrapper.Validator.Rate
 	if err := staking.UpdateValidatorFromEditMsg(&wrapper.Validator, ev); err != nil {
 		return err
 	}
-	wrapper.Validator.UpdateHeight = blockNum
+	newRate := wrapper.Validator.Rate
+	// update the commision rate change height
+	if oldRate.IsNil() || (!newRate.IsNil() && !oldRate.Equal(newRate)) {
```

Comment:
Is the check of MaxChangeRate implemented?

Reply:
Not implemented, see the validity check TODO